lib/cmd/fh3/credential/list.js

```diff
@@ -0,0 +1,19 @@
+/* globals i18n */
+var common = require('../../../common.js');
+module.exports = {
+  'desc' : i18n._('Operations on credential bundles'),
+  'examples' : [{
+    cmd : 'fhc credential list',
+    desc : 'List of all credentials where <bundle-id> is a credentials bundle id'
+  }],
+  'demand' : [],
+  'alias' : {},
+  'describe' : {
+  },
+  'url' : '/box/api/credentials',
+  'method' : 'get',
+  'postCmd': function(response, cb) {
+    response._table = common.createTableForCredentials(response);
```

Comment:
For more readability  I would add alphabetical sorting based on credential bundle name here.